### ISSUE TITLE
is there MAML++?

### ISSUE THREAD

**Issue by brando90** [tristandeleu/pytorch-meta#98]
It seems MAML is sort of unreliable/unstable. Is there a more stable implementation of it like MAML++?

**Comment by tristandeleu** [comment]
This repo does not contain any full implementation of any algorithm. The examples are only meant to showcase the functionalities of Torchmeta. However the library should contain all the tools to built an implementation of MAML++.

For an implementation of MAML, you can check https://github.com/tristandeleu/pytorch-maml which is a better tested implementation of MAML using Torchmeta. It doesn't not have an implementation of MAML++ readily available, but it has a number of features which should make the implementation of MAML++ easy enough (such as learning rate scheduling).
